runtime/elem/Runtime.h

```diff
@@ -354,7 +354,7 @@ namespace elem
             auto ptr = std::dynamic_pointer_cast<RootNode<FloatType>>(it->second);
             if (ptr)
             {
-                ptr->activate(currentRoots.empty() ? FloatType(1) : FloatType(0));
+                ptr->setProperty("active", true);
```

Comment:
Just to make sure I'm following here, even though this was maybe my idea anyways from our early convos– this line does change the default behavior of the initial render(), right? But our intention is that a user who wants to disable the fade on the first render should make their first render call with `renderWithOptions({rootFadeInMs: 0}, ...graph);`

Just want to be clear on that so I can update the docs correctly when we push v4

Reply:
exactly!